### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Stan0934**
- [DEMO LINK](https://stan0934.github.io/layout_stars/)
- [TEST REPORT LINK](https://stan0934.github.io/layout_stars/report/html_report/)

### CODE REVIEW

**Comment by Stan0934** [comment]
BACKSTOP ERROR: Reference file not found /Users/konstantinfedosenko/projects/layout_stars/backstop_data/bitmaps_reference/stars--2.png

BACKSTOP ERROR: Reference file not found /Users/konstantinfedosenko/projects/layout_stars/backstop_data/bitmaps_reference/stars--4.png

BACKSTOP ERROR: Reference file not found /Users/konstantinfedosenko/projects/layout_stars/backstop_data/bitmaps_reference/stars--5.png

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,47 @@
...
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--2">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star s3"></div>
```
[the remark this answers is not in the document]
Comment (Stan0934) [reply src/index.html]: Removed redundant code

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  margin-right: 4px;
+}
+
+.stars--0 div:nth-child(n) {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 div:nth-child(n + 2) {
+  background-image: url(images/star.svg);
+}
```
[the remark this answers is not in the document]
Comment (Stan0934) [reply src/style.css:24]: Should I do it in this task or just get familiar with this function?

**Review by polosanya — CHANGES_REQUESTED** [review]
Try to delete `bitmaps_reference` folder and `npm i` again
image
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review 🙂

**Comment by Stan0934** [comment]
Thanks for the advice! All tests passed. But on the Mate Academy website it is indicated that the Pull request was rejected

**Review by polosanya — APPROVED** [review]
Good job!

**Pull request closed without merging**